src/translations/es.json

```
@@ -216,12 +216,13 @@
     "join": "Unirse"
   },
   "rules": {
-    "rulesTitle": "Reglas de Cuttle",
-    "rulesText": "Cuttle es un juego de cartas de batalla para dos jugadores que se juega con una baraja de cartas estándar de 52 cartas. Tiene el matiz estratégico de los juegos de cartas coleccionables como Magic, con el elegante equilibrio de una baraja estándar... ¡y puedes jugarlo gratis! ¡Pon a prueba tu temple en el juego de cartas más profundo bajo el mar!",
+    "cuttleTitle": "¿Qué es Cuttle",
+    "cuttleText": "Cuttle es un juego de cartas de batalla para dos jugadores que se juega con una baraja estándar de 52 cartas. Tiene la sutileza estratégica de juegos de cartas de intercambio como Magic, con el equilibrio elegante de una baraja estándar, ¡y puedes jugarlo gratis! ¡Pon a prueba tu habilidad en el juego de cartas más profundo bajo el mar!",
     "findGame": "Encuentra un juego",
     "signUp":"Inscríbete para jugar en línea",
-    "tutorialTitle": "Video Tutorial",
-    "tutorialText": "Mira el tutorial oficial de cuttle.cards para aprender las reglas y tener una idea del flujo del juego.",
+    "rulesTitle": "Reglas de Cuttle",
+    "rulesReadText": "Leer le oficial ",
+    "rulesWatchText": " o mira el tutorial en video de Cuttle.cards a continuación para aprender las reglas y familiarizarte con el flujo del juego.",
```

We don't need the whitespace on either side of the link in the text
```suggestion
    "rulesReadText": "Leer le oficial",
    "rulesWatchText": "o mira el tutorial en video de Cuttle.cards a continuación para aprender las reglas y familiarizarte con el flujo del juego.",
```